src/dyad/core/dyad_core.c

```diff
@@ -152,15 +153,16 @@ DYAD_CORE_FUNC_MODS dyad_rc_t publish_via_flux (const dyad_ctx_t* restrict ctx,
         goto publish_done;
     }
     rc = DYAD_RC_OK;
+kvs_transaction_create_and_pack_region_finish:;
```

Comment:
goto target lables should ends with :; instead of : to avoid compiler warning with certain compilers. I added ';'. through out the PR.